src/convertToObject.js

```diff
@@ -13,6 +13,22 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const arrOfString = sourceString.split(';');
+  const result = {};
+
+  arrOfString.filter(item => {
+    if (item) {
+      return item;
+    }
+  }).forEach(item => {
+    const x = item.split(':');
```

Comment:
что значит х?
```suggestion
    const [key, value] = item.split(':');
```
используй деструктуризацию, для того чтобы достать нужные элементы из массива